src/pages/Help/InformationSheetDictionary/InformationSheetDictionary.js

```diff
@@ -14,23 +14,22 @@ import { callCoverCropApi } from '../../../shared/constants';
 
 const InformationSheetDictionary = ({ zone, from }) => {
   // redux vars
-  const filterStateRedux = useSelector((stateRedux) => stateRedux.filterData);
   const apiBaseUrlRedux = useSelector((stateRedux) => stateRedux.sharedData.apiBaseUrl);
 
   // useState vars
   const [dictionary, setDictionary] = useState([]);
-  const section = window.location.href.includes('species-selector') ? 'selector' : 'explorer';
-  const sfilters = filterStateRedux[section];
-
-  const currentZone = zone || sfilters.zone;
+  const stateId = +localStorage.getItem('stateId');
```

Comment:
why are you saving these in local storage?

Reply:
The data Dictionary opens in a new window the Redux info is blank.  I needed to save that info to local storage so that data was still available in the new open window.  This is why the results returned the same every time it kept defaulting to zone 6 no matter what the selection was.